server/lib/fetch.py

```diff
@@ -242,21 +246,29 @@ def series_facet(entities, variables, all_facets):
     },
     "data": {
       <var_dcid>: {
-        <entity_dcid>: {
+        <entity_dcid>: [{
           "facet": <facet_id>,
-          "series": [
-            {
-              "value": <count_of_observations>
-            }
-          ]
-        }
+          "obsCount": <count of observations>,
+          "earliestDate": <earliest date of observations>,
+          "latestDate": <latest date of observations>
+        }, ...]
       }
     }
   }
 
   """
   resp = dc.series_facet(entities, variables)
-  return _compact_series(resp, all_facets)
+  compacted_series = _compact_series(resp, all_facets)
+  processed_series = {'facets': compacted_series.get('facets', {}), 'data': {}}
+  # Update compacted series so that the entity data is always a list.
```

Comment:
Instead of updating here, do you want to update compact_series() function directly?

I feel it's nice to make _compact_series and _compact_point to return uniform response (the origin decision to return either {} or [] is not optimal)

Reply:
This will be a bigger change (need to also update the frontend logic that calls series/point APIs), so added a TODO for this